.github/workflows/selftest-linux.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: ./linux
         with:
           tag: '1.3.0'
-          integration: 'nri-snmp'
+          integration: 'nri-cassandra'
```

Comment:
these are local packages uploaded to the repo i think

Reply:
🤦